packages/sit-onyx/src/components/OnyxSegmentedControl/types.ts

```diff
@@ -1,13 +1,12 @@
 import type { DensityProp } from "../../composables/density.js";
 import type { SkeletonInjected } from "../../composables/useSkeletonState.js";
 import type { BaseSelectOption } from "../../types/components.js";
-import type { Nullable } from "../../types/utils.js";
 
 export type OnyxSegmentedControlProps = DensityProp & {
   /**
    * value of the active control element
    */
-  modelValue?: Nullable<string>;
+  modelValue: string;
```

Comment:
Not related to this PR but I noticed that the emits are not typed correctly for the OnyxSegmentedControl, it should be:

```ts
const emit = defineEmits<{
  /**
   * Emitted when the selected value changes.
   */
  "update:modelValue": [value: string];
}>();
```